src/Symfony/Component/Finder/Finder.php

```diff
@@ -373,6 +374,17 @@ public function ignoreVCS($ignoreVCS)
         return $this;
     }
 
+    public function ignoreVCSIgnored($ignoreVCSIgnored)
```

Comment:
Because we're on php 7.1 now, we should add type-hints to new methods: `bool $ignoreVCSIgnored`.